jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -5,7 +5,12 @@ def run(params) {
         env.resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
         // The junit plugin doesn't affect full paths
         junit_resultdir = "results/${BUILD_NUMBER}/results_junit"
-        env.common_params = "--outputdir ${resultdir} --tf ${params.tf_file} --gitfolder ${resultdir}/sumaform"
+
+        if (!params.terraform_parallelism) {
+            params.terraform_parallelism = 10
+        }
+
+        env.common_params = "--outputdir ${resultdir} --tf ${params.tf_file} --gitfolder ${resultdir}/sumaform --parallelism ${params.terraform_parallelism}"
```

Comment:
If I may suggest...
I would design it differently.

I think the default value must be managed and coming from Terracumber:
https://github.com/uyuni-project/terracumber/pull/11/files#diff-56646f4f9cb011c599f92d8cd92395d82e90023b111c80931c0dcf8d8267bf47R51-R53

And so, better not handle the default value two times.
So, what I would do is something like... (not tested):

```suggestion
        env.common_params = "--outputdir ${resultdir} --tf ${params.tf_file} --gitfolder ${resultdir}/sumaform"

        if (params.terraform_parallelism) {
            env.common_params = "${env.common_params} --parallelism ${params.terraform_parallelism}"
        }
```

(Keep it only as example, don't directly commit the suggestion :D )

Reply:
Yes, the default is coming from terracumber, so I think we can skip the whole check for the parameter so it should always be set there. Either with the default value (10) or the value set by the user.
What do you say @jordimassaguerpla?

Reply:
I would keep the check just in case. We could change the parameters list later on and forget about this. It is to be on the safe side.

Reply:
I like Oscar's suggestion.

Reply:
Not sure if you get the detail of `env.common_params = "${env.common_params} --parallelism ${params.terraform_parallelism}"` the idea is that if the variable is defined, we add the parameter and its value, otherwise we don't add anything and therefore it will pick the default value from terracumber.

That way we don't handle a default value two times, one in Terracumber and another in the Jenkins Pipeline.